soh/soh/GameMenuBar.cpp

```diff
@@ -1121,6 +1121,8 @@ namespace GameMenuBar {
             UIWidgets::Tooltip("Hides the game version and build details in the boot logo start screen");
             UIWidgets::PaddedEnhancementCheckbox("Better Debug Warp Screen", "gBetterDebugWarpScreen", true, false);
             UIWidgets::Tooltip("Optimized debug warp screen, with the added ability to chose entrances and time of day");
+            UIWidgets::PaddedEnhancementCheckbox("Debug Warp Screen Translation", "gDebugWarpScreenTranslation", true, false);
```

Comment:
maybe we should default this to true, just from a "dev experience" stand point having the default warp menu be something i can read is pretty nice